### ISSUE TITLE
Converting single to multisite error: 404 Page Not Found on all pages/all sites

### ISSUE THREAD

**Issue by tlflow** [statamic/cms#1873]
## Bug Description
When trying to convert a single site to a multisite I get a 404 page not found on all pages
## How to Reproduce
1. create a site 
2. make a user
3. run php please multisite
4. make config changes (config/statamic/sites.php)
5. run php artisan cache:clear 
## Extra Detail
Exception Error
![error](https://user-images.githubusercontent.com/415773/83213945-6ea50780-a131-11ea-89f1-2c15aa4d2207.jpg)

## Environment

**Statamic version:** 3.0.0.-beta-{?}

**PHP version:** 7.4.6 using Laragon (Win PC)

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by jonassiewertsen** [comment]
Did you rename your default page?

**Comment by jonassiewertsen** [comment]
```php
'sites' => [

        'default' => [ // don't modify
   ...
```

The default handle can't be modified at the moment. You can rename the name and locale of course (not sure about the url), but the handle needs stay as is.

**Comment by tlflow** [comment]
@jonassiewertsen I left the default the same and only added the new site:

` 
'sites' => [

        'default' => [
            'name' =>  config('app.name'),
            'locale' => 'en_US',
            'url' => '/',
        ],
        'site_two' => [
            'name' => 'Site Two',
            'locale' => 'en_US',
            'url' => '/site-two/',
        ]

    ]
`

**Comment by tlflow** [comment]
Also, going back to into the CP I'm able to make pages and child pages and it seems to be going to the right paths but it just shows 404 Page Not Found

**Comment by johncarter-** [comment]
Hi @tlflow I know it was the weekend and all, but did you fix this?

I'm also using Laragon. I just tried switching Laragon to nginx but that wasn't it.

Also, when I run `php please stache:clear` the entries become detached from the collection - are you getting that too?

**Comment by johncarter-** [comment]
Sorry for hijacking this issue but I'm pretty sure my issue is related, and due to relative paths, for example on this stache controller method:

```php
// vendor\statamic\cms\src\Stache\Stores\CollectionEntriesStore.php
if (Site::hasMultiple()) {
    [$site, $relative] = explode('/', $relative, 2);
    dd($site, $relative); // 
    if (!$this->collection()->sites()->contains($site)) {
        return false;
    }
}
```
Returns:
```
"C:"
"laragon/www/REDACTED/content/collections/pages\da\2020-06-15-1053.REDACTED.md"
```

Not that I know how to fix it...

**Comment by johncarter-** [comment]
See fix #1933

**Comment by tlflow** [comment]
@johncarter- thanks for reaching out! Yes, I was getting the same thing happen when I cleared the stache, as well. Thanks for pointing to #1933 as it at does seem to fix the issue I was having with 404 errors I was getting.
